index.bs

```diff
@@ -3199,6 +3201,143 @@ The [=remote end steps=] with |command parameters| are:
 1. Return a new map matching the <code>ScriptEvaluateResultSuccess</code>
    production, with the <code>result</code> field set to |result|.
 
+#### The script.invoke Command ####  {#command-script-invoke}
+
+The <dfn export for=commands>script.invoke</dfn> command invokes a provided
+function with given arguments in a given realm.
+
+ <code>RealmInfo</code> can be either a realm or a browsing context.
+
+Issue: declare `RemoteValueArgument`.
+
+Issue: declare `LocalValueArgument`.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      ScriptInvokeCommand = {
+        method: "script.invoke",
+        params: ScriptInvokeParameters
+      }
+
+      ScriptInvokeParameters = {
```

Comment:
Should there be a way to specify the `thisObject` for the call?

Reply:
My take is that `this` will be null by default, so adding an optional `thisObject` later will be easy. It's not strictly necessary since one can always pass it as an argument instead, but might be nice to have for parity with other protocols. Fine to defer for me.